### PR TITLE
PR to main

### PR DESCRIPTION
**Pull request by alexlomme**

### CODE REVIEW

File: lines.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ var colour;
 var colourPanel = document.querySelector("#colours-panel");
 var sizePanel = document.querySelector("#dot-size-panel");
 var dotSize;
+var radius = document.querySelector(".dot-size.chosen circle").getAttribute("r");
```
Comment (nikita-ezhkov) [inline lines.js]: I would use `data-` attribute here. Getting attribute of SVG sub-element is not good approach

File: lines.js [hunk 2]
```diff
@@ -16,9 +17,8 @@ button1.addEventListener('click', buttonState, false);
 colourPanel.addEventListener('click', chooseColour, false);
 sizePanel.addEventListener('click', chooseSize, false);
 
-var radius = 1;
-context.strokeStyle = "#000000";
-context.fillStyle = "#000000";
+context.strokeStyle = document.querySelector(".colour.chosen").dataset.colour;
+context.fillStyle = document.querySelector(".colour.chosen").dataset.colour;
```
Comment (nikita-ezhkov) [inline lines.js]: Can not we query element once?

**Review by nikita-ezhkov — COMMENTED** [review]
In general looks really good. Nice work, man. Just a couple little comments.

**Comment by nikita-ezhkov** [comment]
@alexlomme ping me, when you are done please. I will merge it.

**Comment by alexlomme** [comment]
> @alexlomme ping me, when you are done please. I will merge it.

@nikita-ezhkov

**Review by nikita-ezhkov — APPROVED** [review]

**Comment by nikita-ezhkov** [comment]
@alexlomme i can not merge. I have no permissions 😄

**Pull request merged**